src/lib/graph.ts

```diff
@@ -0,0 +1,34 @@
+export async function graphql<T>(
+  url: string,
+  query: string,
+  retryDelay = 500,
+  retryAttempts = 10
+) {
+  try {
+    const result: { data: T } = await fetch(url, {
+      method: 'post',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({
+        query
```

Comment:
I think you're stringifying a string!

Reply:
mm i'm stringifying an object with a `query` string inside, or are you saying something else?